### PR TITLE
Release 1.4.0 into develop

### PR DESCRIPTION
**Pull request by AliSoftware**
New version 1.4.0

### CODE REVIEW

**Comment by mokagio** [comment]
Just a note here on the failing GitHub Actions build because:

```
Unable to find a spec satisfying nokogiri (~> 1.11) in the set. Perhaps the
lockfile is corrupted? Found nokogiri (1.11.7-x86_64-darwin) that did not match
the current platform.
```

I need to learn more about how Bundler 2.2.x manages the platforms, but I think we can _work around_ this by adding the following to `.bundle/config`:

```
BUNDLE_SPECIFIC_PLATFORM: "false"
```

(_We could also ignore this and move on, but isn't it unsatisfying?_)

**Comment by mokagio** [comment]
Opened #293 to address the CI issue.

**Review by mokagio — APPROVED** [review]
LGTM. 

Once #293 gets into the release branch, we should be able to merge and ship.

File: CHANGELOG.md [hunk 1]
```diff
@@ -29,6 +39,9 @@ _None_
 * Fix the way versioning is handled for alphas – i.e. `version.properties` is indexed by flavor name, defaulting to `zalpha` for alphas. [#283]
 * Fixed an issue in `check_translation_progress` where a wrong evaluation of the progress is possible when there are Waiting string in GlotPress.
 
+### Internal Changes
+
+* Opt-out from installing platform-specific gems with Bundler [#293]
```
Comment (AliSoftware) [inline CHANGELOG.md:44]: Merged 293 too quick and didn't realize this was in the wrong place, needs to be under 1.4.0 not 1.3.1

**Pull request merged**